packages/vm/lib/runBlock.ts

```diff
@@ -12,6 +12,13 @@ import type { RunTxResult } from './runTx'
 import type { TxReceipt } from './types'
 import * as DAOConfig from './config/dao_fork_accounts_config.json'
 
+// For backwards compatibility from v5.3.0,
+// TxReceipts are exported. These exports are
+// deprecated and may be removed soon, please
+// update your imports to the new types file.
+import { PreByzantiumTxReceipt, PostByzantiumTxReceipt, EIP2930Receipt } from './types'
+export { PreByzantiumTxReceipt, PostByzantiumTxReceipt, EIP2930Receipt }
```

Comment:
Ah, just seeing, ignore the other comment. Have added an additional note to https://github.com/ethereumjs/ethereumjs-monorepo/issues/1024.

Reply:
(I am so so looking forward towards the next round of breaking releases 😛 , nevertheless think we need to have some patience there)